ansible/roles/software/ckan/catalog/ckan-app/molecule/catalog-next/molecule.yml

```diff
@@ -0,0 +1,77 @@
+---
+dependency:
+  name: galaxy
+  options:
+    role-file: molecule/shared/requirements.yml
+driver:
+  name: docker
+lint:
+  name: yamllint
+  enabled: false
+platforms:
+  - name: ckan-catalog-app-bionic
+    image: ubuntu:bionic
+    # start-stop-daemon doesn't detect tomcat is running in docker and `service
+    # start/stop` fails. SYS_PTRACE is needed.
+    # https://stackoverflow.com/questions/29683231/tomcat7-fail-to-start-inside-ubuntu-docker-container/29686963#29686963
+    capabilities:
+      - SYS_PTRACE
+    groups:
+      - v2
+provisioner:
+  name: ansible
+  inventory:
+    group_vars:
+      all:
+        catalog_ckan_who_ini_secret: e45cfed3-40f1-41c0-8e92-77eda7ddd9f3
+        # https://github.com/DavidWittman/ansible-redis/blob/21b0b6f9030275a2586baf591f322ce3348b2b2d/tasks/install.yml#L9
+        redis_travis_ci: true
+        ckan_catalog_next: true
+        ckan_uses_gunicorn: true
+        app_repo: https://github.com/GSA/catalog.data.gov
+        ckan_production_ini_template: catalog-next/etc_ckan_production.ini.j2
+        catalog_ckan_plugins_default:
```

Comment:
Hmm, having this here will probably come back to bite us. Maybe we just want to add a TODO comment to remove it once CKAN 2.3 is removed, then we'll fall back to the defaults. Just a :thought_balloon:

Reply:
Agreed.  In fact, I think the whole `catalog-next` molecule scenario is probably temporary.